src/ruby/ext/grpc/rb_channel.c

```diff
@@ -28,6 +28,10 @@
 #include <grpc/support/alloc.h>
 #include <grpc/support/log.h>
 #include <grpc/support/time.h>
+#include <grpc/support/port_platform.h>
+#ifdef GPR_SUPPORT_CHANNELS_FROM_FD
```

Comment:
since https://github.com/grpc/grpc/blob/master/src/core/ext/transport/chttp2/client/insecure/channel_create_posix.c#L24 is already handling platform compatiblity, I think we can remove this `ifdef` and unconditionally include the posix header

Reply:
Done.